src/runtime_context/runtime_context.js

```diff
@@ -22,8 +22,8 @@ const WalletsListController = require('../WalletsList/Controllers/WalletsListCon
 const WalletAppCoordinator = require('../WalletAppCoordinator/WalletAppCoordinator')
 const ExceptionAlerting = require('../MainWindow/Controllers/ExceptionAlerting.browser.web.js')
 
-function NewHydratedContext (initialContext_orNilForNew) {
-  const context = initialContext_orNilForNew != null ? initialContext_orNilForNew : {}
+function NewHydratedContext (initialContextOrNil) {
+  const context = initialContextOrNil != null ? initialContextOrNil : {}
```

Comment:
Personal preferenc: I think in general, we should avoid the use of "Nil", as that's not a valid type in Javascript. In instances where a variable may not be set yet, we should prefer "undefined"

Reply:
Agreed

Reply:
Agreed